app/views/edit/deployment-config.html

```diff
@@ -0,0 +1,380 @@
+                                  <span class="input-group-addon">seconds</span>
+                                </span>
+                                <div class="help-block" ng-attr-id="interval">
+                                  Time to wait between polling deployment status after running a pod.
+                                </div>
+                                <div ng-if="form.interval.$invalid && form.interval.$touched" class="has-error">
+                                  <div ng-if="form.interval.$error.number" class="help-block">
+                                    Must be a number.
+                                  </div>
+                                  <div ng-if="form.interval.$error.min" class="help-block">
+                                    Delay can't be negative.
+                                  </div>
+                                  <span ng-if="form.interval.$error.pattern && !form.interval.$error.min" class="help-block">
+                                    Must be a whole number.
+                                  </span>
+                                </div>
+                              </div>
+                            </div>
+
+                            <div class="form-group">
+                              <label for="maxUnavailable">Maximal Unavailable</label>
+                              <div ng-class="{ 'has-error': form.maxUnavailable.$invalid && form.maxUnavailable.$touched }">
+                                <input id="maxUnavailable"
+                                    type="text"
+                                    placeholder="25%"
+                                    name="maxUnavailable"
+                                    ng-model="strategyData[displayedParams].maxUnavailable"
+                                    ng-pattern="/^\d+[%]{0,1}$/"
+                                    min="0"
+                                    select-on-focus
+                                    class="form-control"
+                                    aria-describedby="max-unavailable-help">
+                              </div>
+                              <div class="help-block" id="max-unavailable-help">
+                                Maximum number of pods that can be unavailable during the update. Can be set to either a percentage (e.g., 10%) or an absolute value (e.g., 2).
+                                <a href="{{'rolling_strategy' | helpLink}}" aria-hidden="true" target="_blank"><span class="learn-more-inline">Learn more&nbsp;<i class="fa fa-external-link"></i></span></a>
+                              </div>
+                              <div ng-if="form.maxUnavailable.$invalid && form.maxUnavailable.$touched" class="has-error">
+                                <span ng-if="form.maxUnavailable.$error.pattern && !form.maxUnavailable.$error.min" class="help-block">
+                                  Must be a whole number or percentage.
+                                </span>
```

Comment:
since the input is text, because of the `%` sign that can occur in the input field, we can not check for `min`